genesyscloud/journey_views/resource_genesyscloud_journey_views.go

```diff
@@ -17,6 +18,29 @@ import (
 	"github.com/mypurecloud/platform-client-sdk-go/v143/platformclientv2"
 )
 
+func getAllJourneyViews(ctx context.Context, clientConfig *platformclientv2.Configuration) (resourceExporter.ResourceIDMetaMap, diag.Diagnostics) {
+	resources := make(resourceExporter.ResourceIDMetaMap)
+	proxy := getJourneyViewProxy(clientConfig)
+
+	// Newly created resources often aren't returned unless there's a delay
+	time.Sleep(5 * time.Second)
```

Comment:
I don't think you need this sleep unless you encountered issues without it. I assume you were using the routing_queue resource as a reference but this isn't something we need to do for most resources